ui/temboardui/static/src/components/settings/EnvironmentMigrationDialog.vue

```diff
@@ -0,0 +1,47 @@
+<script type="text/javascript">
```

Comment:
I suggest we start using the composition API since it's already available in VueJS 2.7.

`<script setup>`